frontend/src/components/VisitRouter/VisitForm.js

```diff
@@ -0,0 +1,98 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Grid from "@material-ui/core/Grid"
+import Select from "@material-ui/core/Select"
+import MenuItem from "@material-ui/core/MenuItem"
+import TextField from "@material-ui/core/TextField"
+import InputLabel from "@material-ui/core/InputLabel"
+import FormControl from "@material-ui/core/FormControl"
+import { PrevPointHeading } from "../Typography"
+import { URGENCY_OPTIONS } from "../../constants"
+
+function VisitForm(props) {
+  const { visitInfo, programList, serviceList, handleVisitChange } = props
```

Comment:
if we know this is what we need out of props why not just make it destrcuture in the function
`const VisitForm = ({visitInfo, programList, serviceList, handleVisitChange}) = {...}`